### PR TITLE
Pax to usdp rename

### PR DESCRIPTION
**Pull request by px-ben**

### CODE REVIEW

File: src/tokens/eth/0x8E870D67F660D95d5be530380D0eC0bd388289E1.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  "symbol": "PAX",
-  "name": "Paxos Standard",
+  "symbol": "USDP",
+  "name": "Pax Dollar",
   "type": "ERC20",
   "address": "0x8E870D67F660D95d5be530380D0eC0bd388289E1",
   "ens_address": "",
   "decimals": 18,
-  "website": "https://www.paxos.com/standard/",
+  "website": "https://www.paxos.com/usdp/",
```
Comment (gitteri) [inline src/tokens/eth/0x8E870D67F660D95d5be530380D0eC0bd388289E1.json:8]: we might as well fill in the social information etc below as well. you can use the links in [etherscan](https://etherscan.io/token/0x8e870d67f660d95d5be530380d0ec0bd388289e1)

Comment (px-ben) [reply src/tokens/eth/0x8E870D67F660D95d5be530380D0eC0bd388289E1.json:8]: I noticed you did originally but it was removed in subsequent commits. I'll add it back.

Comment (gitteri) [reply src/tokens/eth/0x8E870D67F660D95d5be530380D0eC0bd388289E1.json:8]: ah interesting let's leave it out then. maybe they don't use it

Comment (px-ben) [reply src/tokens/eth/0x8E870D67F660D95d5be530380D0eC0bd388289E1.json:8]: A few seem to use it but most don't. A quick grep for `facebook` shows 124 non-empty and 5569 empty, and similar for `twitter`.

**Review by gitteri — APPROVED** [review]

**Pull request merged**